app/components/widgets/MediaShare.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+<widget-window v-if="wData" ref="layout" v-model="tabName">
```

Comment:
add :requestState="requestState" :loaded="loaded" here like in other widgets to allow widget to properly handle loading state. That is not looking good however we already have a refactoring-PR for that, but it is not in staging now.

Reply:
@holiber this is Sean's work in media share

Reply:
I don't see this file on staging. If the code in this branch is using this widget and you are going to merge this branch into staging that must be fixed. You can ask @gettinToasty to help you if you're not sure what is going on here. I think that just adding these 2 params will be fine.

Reply:
media share is ready to go afaik just hasn't been reviewed